Mathlib/Analysis/NormedSpace/WithLp.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2023 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Data.Real.ENNReal
+import Mathlib.LinearAlgebra.FiniteDimensional
+
+/-! # The `WithLp` type synonym
+
+`WithLp p V` is the vector space `V`, but with the Lp norm.
+
+This file defines the vector space structure for all types `V`; the norm structure is built for
+different specializations of `V` in downstream files.
+-/
+
+open scoped ENNReal
+set_option linter.uppercaseLean3 false
+
+universe uK uK' uV
+
+/-- A type synony for the given `V`, associated with the L`p` norm. Note that by default this just
+forgets the norm structure on `V`; it is up to downstream users to implement the L`p` norm (for
+instance, on `Prod` and finite `Pi` types). -/
+@[nolint unusedArguments]
+def WithLp (_p : ℝ≥0∞) (V : Type u) : Type u := V
+
+variable (p : ℝ≥0∞) (K : Type uK) (V : Type uV)
```

Comment:
Are you missing a `(K' : Type uK')` here?

Reply:
Yes, and I wish autoimplicits were disabled so that I didn't need a human to tell me that